inji-web/src/components/Home/HomeQuickTip.tsx

```diff
@@ -5,19 +5,18 @@ import { useTranslation } from "react-i18next";
 export const HomeQuickTip: React.FC<HomeQuickTipProps> = (props) => {
     const { t } = useTranslation("HomePage");
     return (
-        <div className="py-4 mb-5 pb-20">
-          <div className="flex flex-col sm:flex-row items-center mx-[4%] sm:mb-3 py-[10%] sm:py-[4%] px-[2%] rounded-xl bg-gradient-to-r from-iw-primary to-iw-secondary sm:w-[80%] sm:ml-auto sm:mr-auto">
-            <span className="w-full sm:w-1/2 text-2xl text-center sm:text-left px-[5%] text-iw-text font-semibold pb-10 sm:pb-0">
+        <div data-testid="HomeQuickTip-Container" className="py-4 mb-5 pb-20">
+          <div data-testid="HomeQuickTip-Content" className="flex flex-col sm:flex-row items-center mx-[4%] sm:mb-3 py-[10%] sm:py-[4%] px-[2%] rounded-xl bg-gradient-to-r from-iw-primary to-iw-secondary sm:w-[80%] sm:ml-auto sm:mr-auto">
+            <span data-testid="HomeQuickTip-Heading" className="w-full sm:w-1/2 text-2xl text-center sm:text-left px-[5%] text-iw-text font-semibold pb-10 sm:pb-0">
               {t("QuickTip.heading")}
             </span>
-            <div className="w-4/5 sm:w-1/6 sm:ml-auto">
-              <PlainButton testId={"HomeBanner-Get-Started"}  onClick={props.onClick}  title={t("QuickTip.buttontext")} 
-              />
+            <div data-testid="HomeQuickTip-ButtonContainer" className="w-4/5 sm:w-1/6 sm:ml-auto">
```

Comment:
Dont add wrapper Div, add the testid for div inside PlainButton, BorderedButton, SolidButton, instead of adding additional wrapper

Reply:
Added already